make install no longer installs opam-installer if opam is not initialised

From a fresh build/install using `make cold` on alpine:

```
src_ext/jbuilder/_build/install/default/bin/jbuilder build  opam-installer.install opam.install
        opam (internal)
[WARNING] Running as root is not recommended
[ERROR] /root/.opam exists, but does not appear to be a valid opam root. Please remove it and use `opam init', or specify a different `--root' argument
./opam-installer --prefix "/usr/local" --mandir "/usr/local/share/man" opam-actual.install
_build/install/default/bin/opam  => /usr/local/bin/opam
```

Only `opam` is present, not `opam-installer` in the installation.  I'm working around it by manually copying the binaries into the container roots.

--- a/Makefile
+++ b/Makefile
@@ -125,6 +125,7 @@ libinstall: $(JBUILDER_DEP) opam-admin.top $(OPAMLIBS:%=installlib-%)
 
 install: opam-actual.install
 	$(OPAMINSTALLER) $(OPAMINSTALLER_FLAGS) $<
+	$(OPAMINSTALLER) $(OPAMINSTALLER_FLAGS) opam-installer.install
 
 libuninstall: $(OPAMLIBS:%=uninstalllib-%)
 	@

